### ISSUE TITLE
Getting "Checks" error for transmission but it appears to connect

### ISSUE THREAD

**Issue by desultory** [karamanolev/WhatManager2#121]
I'm getting this error:

`TransmissionError('Request failed.', error) TransmissionError: Request failed. Original exception: HTTPHandlerError, "HTTPHandlerError 401: Unauthorized" `

in the checks page but it appears to be listing the torrents properly.

I have my transmission server on a different server and I made sure to set the IP, ports, and credentials properly in the config and database but I'm getting that error.  I also made sure to whitelist the WM2 IP in the transmission config and I have a reverse proxy in Apache that connects to it fine.  This may be because I updated transmission to 2.94 but I can't say that with certainty. 

I believe this issue might be partially because in the past I just deleted torrents from the instance using remote or the web gui when they became unregistered but I do not know if that is the issue.  One thing that I also think might be good is if there was a button to delete torrents from the database if they were no longer in transmission.  I was having issues with this in the past and getting the database entries to delete properly was kind of a pain because it involved getting rid of several related entries.

My setup is a bit weird, as I have a transmission instance on one server with a NFS share to the webserver where they both have the same paths to the files.  For example, on the transmissions server the files may be in /mnt/share/wmfiles/groupid/... and they are also in /mnt/share/wmfiles/groupid/... on the webserver because of how I have the NFS share mounts.  

I haven't been using WM2 lately and I periodically check it to make sure it's running well and this check issue seems to be a new issue that has arisen after I updated transmission to 2.94 without changing any other variable.

**Comment by MADindustries** [comment]
Make sure that the default "transmission-daemon" service was not re-enabled during the update and is not currently running. Sometimes when the default one is enabled, it overrides the "transmission-daemon-redacted01" service and conflicts with the web port.

**Comment by desultory** [comment]
I built transmission from source so I know that is not an issue.  I have my transmission instance for WM called "transmission-daemon-wm" and the config was preserved when I built it.

**Comment by MADindustries** [comment]
Are you able to login to the transmission webui(s) via the expected credentials, port, and address?

**Comment by desultory** [comment]
> 
> 
> Are you able to login to the transmission webui(s) via the expected credentials, port, and address?

Yup, I can log in directly when logged into my VPN and I can log in by proxy through a reverse proxy using Apache where my apache server IP is whitelisted on the transmission server.

**Comment by desultory** [comment]
To clarify, I can log in directly from my VPN on the server where the IP is 172.16.81.51:9092/transmission/web/
I can also log in from my public webserver where the webserver is at 172.16.81.80, and I log in from my.domain.tld/tw/web/ and it has the same prompt

WM2 is running at what.mydomain.tld but it's also running on 172.16.81.80 so it is whitelisted the same way the apache reverse proxy is.  If I can connect through the reverse proxy and it's whitelisted correctly, WM2 should also be able to connect too.  I also just whitelisted what.mydomain.tld in the RPC to be sure it's not sending what.mydomain.tld in the headers somewhere but that didn't make a difference in the checks.

Again, it appears to list the torrents correctly but it's just throwing that error in checks.

**Comment by MADindustries** [comment]
Can you use curl from a shell on the WM instance to try and access the transmission page to verify access from that box to the other?
